source/includes/_invoice-payment.md

```diff
@@ -501,9 +307,7 @@ class InvoicePayment {
    "auditLogs":[]
 }
 ```
```

Comment:
The response looks wrong to me, I think an empty body is sent with the location header. Could you please confirm and make changes as required?